index.js

```diff
@@ -268,6 +274,28 @@ function int() {
   };
 }
 
+function byte() {
```

Comment:
Could we move these up so they match the order at the top? (byte, short, then int)